miqa/learning/nn_classifier.py

```diff
@@ -260,7 +260,7 @@ def evaluate_model(model, data_loader, device, writer, epoch, run_name):
             return outputs
 
 
-def evaluate1(model_path, image_path):
+def evaluate1(model, image_path):
```

Comment:
The invocation in "main" function should be changed too. It currently reads `evaluate1(args.modelfile, args.evaluate1)`

Reply:
Good catch. Updated.